packages/react-dom/src/events/ReactBrowserEventEmitter.js

```diff
@@ -124,25 +125,33 @@ function getListeningForDocument(mountAt: any) {
 export function listenTo(
   registrationName: string,
   mountAt: Document | Element,
+  element: Element,
 ) {
-  const isListening = getListeningForDocument(mountAt);
+  const mountAtListeners = getListenerTrackingFor(mountAt);
   const dependencies = registrationNameDependencies[registrationName];
 
   for (let i = 0; i < dependencies.length; i++) {
     const dependency = dependencies[i];
-    if (!(isListening.hasOwnProperty(dependency) && isListening[dependency])) {
+
+    if (!mountAtListeners.hasOwnProperty(dependency)) {
       switch (dependency) {
         case TOP_SCROLL:
-          trapCapturedEvent(TOP_SCROLL, mountAt);
-          break;
+        case TOP_WHEEL:
```

Comment:
I used to include touch events, but I've reduced the event list for this fix down to wheel and scroll to reduce the testing surface area. If this approach goes well, we could follow up with a PR that evaluates those events too.